src/site/components/service_address.drupal.liquid

```diff
@@ -0,0 +1,85 @@
+<h3>Location and contact information</h3>
+<div class="va-address-block">
+  {% if address.fieldUseFacilityAddress == false %}
+  {% assign address = single.fieldServiceLocationAddress.entity %}
+  {% assign useParent = false %}
+  {% else %}
+  {% assign address = fieldAddress %}
+  {% assign useParent = true %}
+  {% endif %}
+  {% if single.fieldServiceLocationAddress.entity.fieldClinicName%}<p
+    class="vads-u-margin-bottom--0">
+    <strong>{{ single.fieldServiceLocationAddress.entity.fieldClinicName }}</strong>
+  </p>{% endif %}
+
+  {% if address.fieldAddress.addressLine1 and useParent == false %}<p
+    class="vads-u-margin-bottom--0">
+    {{ address.fieldAddress.addressLine1 }}</p>
+  {% else %}
+  <p class="vads-u-margin-bottom--0">
+    {{ fieldAddress.addressLine1 }}</p>
+  {% endif %} {% if address.fieldAddress.addressLine1 and useParent == false %}
+  <p class="vads-u-margin-bottom--0">
+    {{ address.fieldAddress.addressLine1 }}</p>
+  {% else %}
+  <p class="vads-u-margin-bottom--0">
+    {{ fieldAddress.addressLine1 }}</p>
+  {% endif %}
+  {% if address.fieldAddress.addressLine1 and useParent == false %}<p
+    class="vads-u-margin-bottom--0">
+    {{ address.fieldAddress.addressLine1 }}</p>
+  {% else %}
+  <p class="vads-u-margin-bottom--0">
+    {{ fieldAddress.addressLine1 }}</p>
+  {% endif %}
```

Comment:
This address and `useParent ` logic is confusing. Isn't the condition on line 21 the same as line 28?